dds_cli/motd_manager.py

```diff
@@ -63,11 +63,22 @@ def __init__(
     def add_new_motd(self, message):
         """Add a new motd."""
         response_json, _ = dds_cli.utils.perform_request(
-            endpoint=DDSEndpoint.ADD_NEW_MOTD,
+            endpoint=DDSEndpoint.MOTD,
             headers=self.token,
             method="post",
             json={"message": message},
             error_message="Failed adding a new MOTD",
         )
 
         LOG.info("A new MOTD was added to the database")
+
+    def get_latest_motd(self):
+        """Get the latest MOTD from dabase"""
+        response_json, _ = dds_cli.utils.perform_request(
+            endpoint=DDSEndpoint.MOTD,
+            headers=self.token,
+            method="get",
+            error_message="Failed getting MOTD",
+        )
+
+        return response_json
```

Comment:
This is a continuation of the previous comment
```suggestion
        try:
            response_json, _ = dds_cli.utils.perform_request(
                endpoint=DDSEndpoint.MOTD,
                headers=self.token,
                method="get",
                error_message="Failed getting MOTD",
            )
        except: 
            pass
        return response_json.get("message")
```

Reply:
Reason for this btw: If something happens with specifically MOTD Endpoint it should still be ppossible to continue the delivery

Reply:
aha, you mean in case the command is for data upload